tests/google_sheets/test_google_sheets.py

```diff
@@ -328,3 +330,20 @@ def test_default_format():
     value = 44303  # Example serial number representing a date
     result = parse_cell_value(value)
     assert result == value
+
+
+def test_can_instantiate_without_retrieve_token_callback():
+    gsheet_connector = GoogleSheetsConnector(
+        name="test_connector",
+        auth_id="test_auth_id",
+    )
+    assert gsheet_connector
```

Comment:
Could you make an assertion on the fields here ? For example on the token callback ?